Makefile

```diff
@@ -1,9 +1,16 @@
 
 # Image URL to use all building/pushing image targets
-TAG ?= latest
-IMG ?= kudobuilder/controller:${TAG}
+DOCKER_TAG ?= $(shell git rev-parse HEAD)
```

Comment:
That just means `latest` won't be updated anymore, so we should be careful not to refer to it...

Reply:
Yep, my motivation was to address your concern from https://github.com/kudobuilder/kudo/pull/60#discussion_r240084616 essentially to avoid `latest` and have more specific tags.

Reply:
Well to correct myself, Imo `latest` will still be updated. See https://github.com/kudobuilder/kudo/pull/166/files#diff-b67911656ef5d18c4ae36cb6741b7965R192